### DISCUSSION TITLE
Preferred way to stub license API call in test suite

### DISCUSSION THREAD

**Discussion by schmidtg** [avo-hq/avo#862] [Help]
Hi there,
Is there a preferred/suggested way to stub HTTP requests to the license API in the test suite?

I'm looking to open a small PR fix for https://github.com/avo-hq/avo/issues/861 and was getting the following on lots of unit tests after running `bin/test unit`. 

```
  86) SelectField enum hash display_value: true edit with stage
      Failure/Error: HTTParty.post ENDPOINT, body: payload.to_json, headers: {'Content-type': "application/json"}, timeout: REQUEST_TIMEOUT
...
        You can stub this request with the following snippet:

        stub_request(:post, "https://avohq.io/api/v1/licenses/check").
...
...
        stub_request(:post, "https://avohq.io/api/v1/licenses/check").
          with(
            body: hash_including({"license"=>"pro", "license_key"=>"license_123"}))
```

Should we just ignore or disable for the test suite? I'm on the community edition (for now).

**Comment by adrianthedev** [comment]
Hey, sorry for the delay. Have you managed to fix this and get it working? I think all you need to do is create the `.env.test` file `cp spec/dummy/.env.test.sample spec/dummy/.env.test`
